dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -259,12 +288,12 @@ export function AppTable(props: AppTableProps) {
                 <EuiFlexGroup justifyContent="center">
                   <EuiFlexItem grow={false}>
                     <EuiButton fullWidth={false} href={`#/application_analytics/create`}>
-                      Create application
+                      {createButtonText}
                     </EuiButton>
                   </EuiFlexItem>
-                  <EuiFlexItem grow={false}>
+                  {/* <EuiFlexItem grow={false}>
                     <EuiButton fullWidth={false}>Add sample applications</EuiButton>
-                  </EuiFlexItem>
+                  </EuiFlexItem> */}
```

Comment:
should we remove this comment?

Reply:
I will be adding sample applications later, so I kept it.